packages/runtime-core/__tests__/errorHandling.spec.ts

```diff
@@ -53,6 +55,7 @@ describe('error handling', () => {
 
   test('propagation stoppage', () => {
```

Comment:
This test is not about KeepAlive, it's about general error propagation stoppage. I think we should have a separate test for this Otherwise, unrelated  bugs in KeepAlive could bleed into this test.

Reply:
Okay, doing:smile: